ui/app/models/pki/tidy.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+
+@withFormFields()
+export default class PkiTidyModel extends Model {
+  // This model uses the backend value as the model ID
+  get useOpenAPI() {
+    return true;
+  }
+
+  @attr('boolean', { defaultValue: false }) tidyCertStore;
+  @attr('boolean', { defaultValue: false }) tidyRevocationQueue;
+  @attr('string') safetyBuffer;
```

Comment:
should we add a `defaultValue: '72h'` here? I think that might pre-fill the ttl input for the user (instead of waiting for the backend to set the value if the user leaves the input empty)

Reply:
@hellobontempo That makes sense! I wasn't sure if we wanted to add that as a `defaultValue`. I did notice our old model has '72h' as a default. Generally, do we set TTL's to be 72h's? Are defaults usually specified somewhere?

Reply:
usually we can find the default values in the [docs](https://developer.hashicorp.com/vault/api-docs/secret/pki#safety_buffer) but sometimes I run the CURL command in the cli (or smoke test in the UI) to double check the docs are correct